scripts/deploy.sh

```
@@ -352,6 +363,34 @@ main() {
+    else
+      log_info "  Installing controller (CRDs, RBAC, deployment, default-deny policy)..."
+      "$controller_dir/scripts/install-maas-controller.sh" "$NAMESPACE"
+
+      log_info "  Waiting for maas-controller to be ready..."
+      kubectl rollout status deployment/maas-controller -n "$NAMESPACE" --timeout=120s 2>/dev/null || true
+
+      log_info "  Disabling old gateway-auth-policy (replaced by per-route policies)..."
+      NAMESPACE="$NAMESPACE" "$controller_dir/hack/
+      
+      -
+      -gateway-auth-policy.sh"
```

Shellcheck (0.11.0)

[error] This is interpreted as a command name containing a linefeed. Double check syntax.

(SC2289)